Sources/GithubData.swift

```diff
@@ -26,10 +26,13 @@ public class GithubData : JSONConvertibleObject, CustomStringConvertible {
   var issueData: IssueData?
   var changes: Changes?
   var projectCard: ProjectCard?
+  var project: Project?
   var sender: String?
+  var url: String?
```

Comment:
Instead of all these properties maybe a small type?

Reply:
can you elaborate, what does a small type mean?

Reply:
LGTM